.circleci/config.yml

```diff
@@ -47,10 +47,9 @@ commands:
             git checkout -b "$branch_name"
```

Comment:
Also `scripts/enforce-change-notes-updated-on-pr-to-master` + Dockerfile to be patched